otter/tap/api.py

```diff
@@ -286,6 +289,21 @@ def on_client_ready(_):
     return s
 
 
+def setup_converger(parent, kz_client, dispatcher):
+    """Create a Converger service."""
```

Comment:
@cyli Yeah. Since partitioner is child of converger, it should stop when converger stops. Converger should stop when `parent` stops.

@radix Non-blocking: On a different note, I feel `Converger` should take partitioner arguments and create the partitioner internally. This way, if converger implementation ever changes to not use Partitoner, it should be easy to do that. This is definitely non-blocking :)